src/h2_stream.erl

```diff
@@ -114,10 +114,10 @@ start_link(StreamId, Connection, CallbackModule, Socket) ->
 send_pp(Pid, Headers) ->
     gen_fsm:send_event(Pid, {send_pp, Headers}).
 
--spec send_frame(pid(), http2_frame:frame()) ->
+-spec send_data(pid(), http2_frame:frame()) ->
```

Comment:
maybe just a comment, then? "this can only send data frames"

On Thu, May 5, 2016 at 3:13 PM Joe DeVivo <notifications@github.com> wrote:

> In src/h2_stream.erl
> <https://github.com/joedevivo/chatterbox/pull/75#discussion_r62238560>:
>
> > @@ -114,10 +114,10 @@ start_link(StreamId, Connection, CallbackModule, Socket) ->
> >  send_pp(Pid, Headers) ->
> >      gen_fsm:send_event(Pid, {send_pp, Headers}).
> >
> > --spec send_frame(pid(), http2_frame:frame()) ->
> > +-spec send_data(pid(), http2_frame:frame()) ->
>
> there isn't actually a data frame type. we could do {frame_header(),
> h2_frame_data:payload()} but I'm not sure it's worth it
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/joedevivo/chatterbox/pull/75/files/3db4a22e8e66fb5da11a1eed8dfa17f8f575945d#r62238560>
>